proposals/forms.py

```diff
@@ -275,11 +275,6 @@ class Meta:
         widgets = {
             'is_revision': forms.HiddenInput(),
         }
-        error_messages = {
-            'title': {
-                'unique': _('Er bestaat al een aanvraag met deze titel.'),
-            },
-        }
```

Comment:
Dropping the unique check sure does solve the issue easily, but it's kinda drastic.

This is a big functional change that should be discussed with the secretary first.

Especially as there's a pretty easy fix (albeit less easy than this fix...) that does not involve doing this...

Reply:
Nooo, it's me that's being an idiot... Sorry!

I forgot we changed that lol. Your solution is correct.